pkg/webhooks/namespace/namespace.go

```diff
@@ -123,39 +168,40 @@ func (s *NamespaceWebhook) Authorized(request admissionctl.Request) admissionctl
 // Is the request authorized?
 func (s *NamespaceWebhook) authorized(request admissionctl.Request) admissionctl.Response {
 	var ret admissionctl.Response
+
+	// Picking OldObject or Object will suffice for most validation concerns
 	ns, err := s.renderNamespace(request)
```

Comment:
considering this is the only place we use `renderNamespace`, I wonder if it would be helpful to combine the logic of `renderNamespace` and `renderOldAndNewNamespaces` and use the resulting function in all places (or if that would, instead, be more confusing?)

Reply:
I actually had it combined in a previous (uncommitted) but the logic and utilization ended up being less readable. The method had three returns, and there still had to be logic at this point in the package to know which to use.

I'm certain there is room for more refactoring, but I'm not quite sure it includes combining the logic of both.

Reply:
nod, I figured the number of returns would make for some difficult reading and logic. this is good (for now :) )